pagopa/io-app
# chore: [IOBP-795] Add payment method onboarding contextual help

## Short description
This PR adds contextual help to the payment onboarding screen

## List of changes proposed in this pull request
- Replaced the deprecated `RNWithLargeScrollHeader` with the new `IOScrollViewWithLargeHeader`
- Added the `showHelp` property and `faqCategories`

## How to test
When you start a payment method onboarding, you should be able to see the contextual help in the `PAYMENT_ONBOARDING_SELECT_METHOD` screen
